packages/core-data/src/actions.js

```diff
@@ -554,6 +554,23 @@ export const saveEntityRecord =
 		const entityIdKey = entityConfig.key || DEFAULT_ENTITY_KEY;
 		const recordId = record[ entityIdKey ];
 
+		// For back-compat, we allow querying for static templates through
+		// wp_template.
+		if (
+			kind === 'postType' &&
+			name === 'wp_template' &&
+			typeof recordId === 'string' &&
+			! /^\d+$/.test( recordId )
+		) {
+			name = '_old_wp_template';
+			entityConfig = configs.find(
+				( config ) => config.kind === kind && config.name === name
+			);
+			if ( ! entityConfig ) {
+				return;
+			}
+		}
```

Comment:
Can we handle this at the entity config level? Similar exceptions can pile up and feel very hacky.

P.S. We might want to reconsider REST API changes; currently, they are breaking changes, which don't align with WP philosophy.

Reply:
There's no REST API changes?

Reply:
Well, actually one change we need to make to the old /templates endpoint is activate the template if anymore makes an edit to keep 100% the same behavior.

Reply:
Btw, this PR was a quick draft, I realized that it's completely not what we want, what we should do instead is create a template through the normal entity and make it active, rather than call the old endpoint. It's going to be way easier than having to do some custom receiving of entities of this call.